backend/test/api-test-example.test.js

```diff
@@ -0,0 +1,49 @@
+const supertest = require("supertest");
+const app = require("../app");
+const request = supertest(app);
+const dbHandler = require("./db-handler");
+
+const Event = require("../models/event.model.js");
+
```

Comment:
At the root of the backend test folder are examples for setting up the tests, and the file for setting up and tearing down the database.